### PR TITLE
adding table with some models to readme

### PR DESCRIPTION
**Pull request by okuchaiev**
Signed-off-by: Oleksii Kuchaiev <okuchaiev@nvidia.com>

### CODE REVIEW

**Comment by okuchaiev** [comment]
@yzhang123 are BERT links correct?
@redoctopus is QuartzNet link correct?
@blisc are TTS links correct?

**Comment by blisc** [comment]
Your table doesn't render in the new readme

File: README.rst [hunk 1]
```diff
@@ -87,6 +87,45 @@ See `examples/start_here` to get started with the simplest example. The folder `
...
+| ASR        | `QuartzNet15x5Zh <https://ngc.nvidia.com/catalog/models/nvidia:aishell2_quartznet15x5>`_     | AI-shell2 Mandarin    |
+|            |                                                                                              |                       |
+|            |                                                                                              |                       |
+|            |                                                                                              |                       |
++------------+----------------------------------------------------------------------------------------------+-----------------------+
+| NLP        | `BERT base uncased <https://ngc.nvidia.com/catalog/models/nvidia:bertbaseuncasedfornemo>`_   |English Wikipedia and  |
+|            |                                                                                              |BookCorpus dataset     |
+|            |                                                                                              |seq len <= 128         |
```
Comment (yzhang123) [inline README.rst]: please change seq len to 512

**Comment by yzhang123** [comment]
> @yzhang123 are BERT links correct?

we also have a bert cased model, but the accuracy is not on par with google's.

**Comment by okuchaiev** [comment]
@blisc please try again (pushed update)

**Comment by blisc** [comment]
Do we care about the LibriSpeech checkpoints? The Jasper checkpoints? Speech Commands?
Ie, everything in this google sheet: https://docs.google.com/spreadsheets/d/12DdHYXjWgKKyWE7gm9h_VQ_NLNWVilmEL7xHdLw630c/edit?usp=sharing

File: README.rst [hunk 2]
```diff
@@ -66,27 +67,70 @@ THE LATEST STABLE VERSION OF NeMo is **0.9.0** (Available via PIP).
...
-See `examples/start_here` to get started with the simplest example. The folder `examples` contains several examples to get you started with various tasks in NLP and ASR.
+See `examples/start_here` to get started with the simplest example.
 
 **Tutorials**
 
 * `Speech recognition <https://nvidia.github.io/NeMo/asr/intro.html>`_
 * `Natural language processing <https://nvidia.github.io/NeMo/nlp/intro.html>`_
 * `Speech Synthesis <https://nvidia.github.io/NeMo/tts/intro.html>`_
 
+Fine-tuning using NeMo
```
Comment (blisc) [inline README.rst]: I wouldn't put the TTS models under "Fine-tuning". You can't easily fine-tune Tacotron and you do not need to fine-tune Waveglow

Comment (blisc) [reply README.rst]: Just call this section pre-trained models

File: README.rst (hunk 1 above)
Comment (okuchaiev) [reply README.rst]: fixed

File: README.rst (hunk 2 above)
Comment (okuchaiev) [reply README.rst]: makes sense - will do

Comment (okuchaiev) [reply README.rst]: done

File: docs/docs_zh/sources/source/index.rst [hunk 3]
```diff
@@ -65,15 +65,11 @@ Neural Modules (NeMo) 是一个用神经模块来构建 AI 应用的工具包，
...
+    docker pull nvcr.io/nvidia/nemo:v0.10
 
-    # 运行下面两个命令之一
-    # 如果你的 docker 版本 <19.03
-    nvidia-docker run -it --rm --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/nemo:v0.9
-
-    # 如果你的 docker 版本 >=19.03
-    docker run -it --rm --gpus all --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/nemo:v0.9
+    # Run Docker for docker version >=19.03
+    docker run --runtime=nvidia -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/nemo:v0.10
```
Comment (yzhang123) [inline docs/docs_zh/sources/source/index.rst:72]: i have >19.03 and i get error with --runtime=nvidia, it only works with --gpus all

File: README.rst [hunk 4]
```diff
@@ -55,38 +56,75 @@ The toolkit comes with extendable collections of pre-built modules for automatic
...
- NVIDIA `NGC NeMo Toolkit container <https://ngc.nvidia.com/catalog/containers/nvidia:nemo>`_ is now available.
 
-* Pull the docker: ``docker pull nvcr.io/nvidia/nemo:v0.9``
-* Run: ``docker run --runtime=nvidia -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/nemo:v0.9``
 
 If you are using the NVIDIA `NGC PyTorch container <https://ngc.nvidia.com/catalog/containers/nvidia:pytorch>`_ follow these instructions
 
 * Pull the docker: ``docker pull nvcr.io/nvidia/pytorch:20.01-py3``
-* Run: ``docker run --runtime=nvidia -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/pytorch:20.01-py3``
+* Run:``docker run --runtime=nvidia -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/pytorch:20.01-py3``
```
Comment (yzhang123) [inline README.rst:72]: same here, --runtime=nvidia is deprecated

File: docs/sources/source/index.rst [hunk 5]
```diff
@@ -66,19 +66,15 @@ You can use NVIDIA `NGC NeMo container <https://ngc.nvidia.com/catalog/container
...
-    docker pull nvcr.io/nvidia/nemo:v0.9
-
-    # Do one of the two following commands
-    # Run Docker for docker version <19.03
-    nvidia-docker run -it --rm --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/nemo:v0.9
+    docker pull nvcr.io/nvidia/nemo:v0.10
     
     # Run Docker for docker version >=19.03
-    docker run -it --rm --gpus all --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/nemo:v0.9
+    docker run --runtime=nvidia -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/nemo:v0.10
```
Comment (yzhang123) [inline docs/sources/source/index.rst:72]: same here

File: docs/docs_zh/sources/source/index.rst (hunk 3 above)
Comment (okuchaiev) [reply docs/docs_zh/sources/source/index.rst]: changed

File: docs/sources/source/index.rst (hunk 5 above)
Comment (okuchaiev) [reply docs/sources/source/index.rst]: changed

File: README.rst (hunk 4 above)
Comment (okuchaiev) [reply README.rst]: changed
